### PR TITLE
cider-2: init at 2.5.0

### PR DESCRIPTION
**Pull request by itsvic-dev**
Cider 2 is a third-party client for Apple Music, available for purchase via itch.io and Microsoft Store (Windows-only).
https://cider.sh/

(sorry about accidentally touching the formatting of another maintainer's keys, I have autoformatting enabled in my code editor)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by itsvic-dev** [comment]
moved the package definition to pkgs/by-name as suggested by nixpkgs-vet.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3302,9 +3302,7 @@
     github = "cageyv";
     githubId = 51059484;
     name = "Vladmir Samoylov";
-    keys = [
-      { fingerprint = "8916 F727 734E 77AB 437F  A33A 19AB 76F5 CEE1 1392"; }
-    ];
+    keys = [ { fingerprint = "8916 F727 734E 77AB 437F  A33A 19AB 76F5 CEE1 1392"; } ];
```
Comment (Aleksanaa) [inline maintainers/maintainer-list.nix:3305]: Please don't apply format to this file

File: pkgs/by-name/ci/cider-2/package.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      cp -r ${contents}/usr/share/icons $out/share
+    '';
+
+  meta = with lib; {
+    description = "Cider is a powerful music player that allows you listen to your favorite tracks with style";
+    homepage = "https://cider.sh";
+    license = licenses.unfree;
+    mainProgram = pname;
+    maintainers = [ maintainers.itsvic-dev ];
+    platforms = [ "x86_64-linux" ];
```
Comment (Aleksanaa) [inline pkgs/by-name/ci/cider-2/package.nix:45]:
```suggestion
  meta = {
    description = "Powerful music player that allows you listen to your favorite tracks with style";
    homepage = "https://cider.sh";
    license = lib.licenses.unfree;
    mainProgram = "cider-2";
    maintainers = with lib.maintainers; [ itsvic-dev ];
    platforms = [ "x86_64-linux" ];
```

**Comment by itsvic-dev** [comment]
applied the suggestions and rebased to latest master (71ccc3b0b040a68100275f497e264c3abf373f58)

**Comment by Aleksanaa** [comment]
I'd assume it works on your end. Please verify it still works.

**Comment by itsvic-dev** [comment]
indeed it does still work, resulting in store path `/nix/store/m8dm83kbqy60kzklvfbrxwxnzk805bw3-cider-2-2.5.0`

**Pull request merged**